lib/actions.js

```diff
@@ -58,6 +66,64 @@ const _createAccount = (walletId, accountId, accountInfo) => ({
   type: WALLET_CREATE_ACCOUNT,
   payload: { walletId, accountId, accountInfo }
 });
+// multisig wallet helpers
+const _getMultisigWallets = (multisigWallets) => ({
```

Comment:
this should probably be called `setMultisigWallets` since that's the action that is being returned.